recipes-core/images/wpe-initramfs-image.bb

```diff
@@ -5,5 +5,3 @@ DESCRIPTION = "WPE initramfs rootfs image"
 LICENSE = "MIT"
```

Comment:
This file and all images should be moved to the meta-metrological-rpi

Reply:
Do we need to move all image files ? then we have to create this image file for each platform right ?

Reply:
Correct. No one exept rpi builds use this. meta-wpe should be platform independent, so please remove them here and add them to the meta-metrological-raspberrypi

Reply:
removed from this layer. it was already added to ml rpi layer